### ISSUE TITLE
Intro Part 3 doesn't work properly

### ISSUE THREAD

**Issue by albka1986** [googlecodelabs/exoplayer-intro#25]
`Play video!
Know what? We are done with this as well. Change the uri in the initializePlayer to R.string.media_url_mp4 instead of R.string.media_url_mp3, start the app again and test the behaviour after being backgrounded with video playback as well:

Uri uri = Uri.parse(getString(R.string.media_url_mp4));
The PlayerView does it all. Instead of the artwork the video is rendered full screen.

You rock! You just managed to have a rock solid full screen activity for proper media streaming on Android!`

This part doesn't work for me. App runs but without voice & video. I see buttons but a screen is just black When I run mp3 everything is ok. A link of the mp4 is ok, I checked in Google Chrome.

**Comment by e-kuri** [comment]
Same problem here. Video is not reproduced and the artwork is not displayed when reproducing the mp3 file.

**Comment by PabloRodrizUnileon** [comment]
@albka1986 and @e-kuri . The solution for me was to edit the `R.string.media_url_mp4` resource so it uses `https` instead of `http`. I arrived to this when i checked on chrome and i realized that the resource is in http in Android Studio, but Chrome redirects to https.

Once that was fixed, it wasn't resuming the video properly as stated in these Issues:
[https://github.com/googlecodelabs/exoplayer-intro/issues/26](url)
[https://github.com/googlecodelabs/exoplayer-intro/issues/19](url)

but the solution is there also.
